### PR TITLE
Specification for xcube Multi-Resolution Datasets

### PR DESCRIPTION
**Pull request by forman**
First version of a specification for xcube Multi-Resolution Datasets.

### CODE REVIEW

File: docs/source/mldatasets.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
+xcube Multi-Resolution Datasets
+===============================
+
+Definition
+----------
+
+A xcube _multi-resolution dataset_ refers to an N-D [image
```
Comment (TonioF) [inline docs/source/mldatasets.md:7]:
```suggestion
A xcube _multi-resolution dataset_ uses an N-D [image 
```

File: docs/source/mldatasets.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+Definition
+----------
+
+A xcube _multi-resolution dataset_ refers to an N-D [image 
+pyramid](https://en.wikipedia.org/wiki/Pyramid_(image_processing)) 
+where an _image_ refers to a 2-D dataset with two spatial dimensions
+in some horizontal coordinate system.
+
+A multi-resolution dataset comprises a fixed number of
+_levels_, which are regular datasets at different spatial resolutions.
```
Comment (TonioF) [inline docs/source/mldatasets.md:13]:
```suggestion
_levels_, which are regular datasets covering the same spatial area at different resolutions.
```

**Review by TonioF — COMMENTED** [review]
Some suggestions.
Also, I take this as a an opportunity to remind of https://github.com/dcs4cop/xcube/pull/493

**Review by AliceBalfanz — APPROVED** [review]

File: docs/source/mldatasets.md (hunk 1 above)
Comment (forman) [reply docs/source/mldatasets.md:7]: That is not correct.

**Pull request merged**